### PR TITLE
Hub 01 intercom dialogue refinement, October edition

### PR DESCRIPTION
**Pull request by Ilysen**
#### Summary
Content "Touched up some of the intercom's dialogue"

#### Purpose of change

In the past PRs of touching up dialogue, I've written a buncha stuff about how standardization and good grammar benefits immersion and believability, and that's still the case here. ;P

#### Describe the solution

* Rewrote a bunch of the intercom's dialogue for grammar and tonal consistency. In general, they are less chummy and more clinical, and especially odd and suspicious when speaking about the Exodii.
* After the exotic barrel is researched, the Hub will start selling new 60-round box magazines for 12.3 ln, in addition to the current behavior of being able to slot in standard ones. Two of these are also provided with the conversion kit itself.
* Added more branching dialogue to a few topics, including the exotic barrel research (where the response is different if you've already met with Rubik) and the HWP requisition.
* You can now provide a PS md. 71z and its magazine to the intercom, instead of a PA md. 68 and its magazine. The old option is still there; now you can just use either rifle if you don't have access to the other.

#### Describe alternatives you've considered

Nothing major, but I've thought significantly of how to tackle the way dialogue is handled right now. There's no kind of standardization of dialogue and variable IDs and formatting, and I'm really not sure what the most elegant way to handle it is. Progress continues.

#### Testing

Talked a whole bunch with the intercom, using quicksaves and quickloads to go through dialogue at certain points. I made sure that all the new dialogue looked okay and I thiiink I got the typos out.

#### Additional context

cataclysm-tiles_GgTupDHjpt
cataclysm-tiles_8xUWyD3lL7

### CODE REVIEW

File: data/json/npcs/robofac/robofac_intercom/robofac_intercom_prototypes.json [hunk 1]
```diff
@@ -579,24 +635,83 @@
...
+        "condition": {
+          "and": [
+            { "u_has_items": { "item": "pamd71z", "count": 1 } },
+            { "u_has_items": { "item": "exodiiminimag", "count": 1 } },
+            { "u_has_items": { "item": "123ln", "count": 5 } }
+          ]
+        },
+        "effect": [
+          { "u_consume_item": "pamd68", "count": 1, "popup": true },
+          { "u_consume_item": "exodiiBRmag", "count": 1, "popup": true },
```
Comment (andrei8l) [inline data/json/npcs/robofac/robofac_intercom/robofac_intercom_prototypes.json:653]:
```suggestion
          { "u_consume_item": "pamd71z", "count": 1, "popup": true },
          { "u_consume_item": "exodiiminimag", "count": 1, "popup": true },
```

File: data/json/npcs/robofac/robofac_intercom/robofac_intercom_prototypes.json [hunk 2]
```diff
@@ -346,18 +355,65 @@
...
   },
+  {
+    "id": "TALK_ROBOFAC_INTERCOM_RIFLE_ADAPTERS",
+    "type": "talk_topic",
+    "dynamic_line": "Adapters are standalone conversion kits, and can be swapped in the field without any tools, though it will take a minute or so.  The factory standard includes adapters for the following calibers… 5.56 NATO, 7.62 NATO, 9mm Parabellum, and 12-gauge shotshell.  Two loaded magazines are also provided for each of these calibers, and additional empty magazines can be purchased through trade.  A maintenance kit with the necessary tools to service the Platform is also provided."
+  },
+  {
+    "id": "TALK_ROBOFAC_INTERCOM_RIFLE_TIMEGATE",
+    "type": "talk_topic",
+    "dynamic_line": "As before, each Platform is made to order. This is a slow process for several reasons.  First, the parts must be printed to spec; this takes a great degree of precision, and so the process is lengthy.  The remainder of the week is spent running reliability tests as each conversion kit finishes production.  I understand that it isn't an ideal option, but I believe the quality of the product is worth that investment."
```
Comment (MNG-cataclysm) [inline data/json/npcs/robofac/robofac_intercom/robofac_intercom_prototypes.json:385]:
```suggestion
    "dynamic_line": "As before, each Platform is made to order.  This is a slow process for several reasons.  First, the parts must be printed to spec; this takes a great degree of precision, and so the process is lengthy.  The remainder of the week is spent running reliability tests as each conversion kit finishes production.  I understand that it isn't an ideal option, but I believe the quality of the product is worth that investment."
```

Comment (MNG-cataclysm) [reply data/json/npcs/robofac/robofac_intercom/robofac_intercom_prototypes.json:385]: Spacing fix

**Review by MNG-cataclysm — CHANGES_REQUESTED** [review]

File: data/json/items/gun/robofac_gun.json [hunk 3]
```diff
@@ -424,5 +424,23 @@
     "ammo_type": [ "223" ],
     "flags": [ "MAG_BULKY" ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "223": 100 } } ]
+  },
+  {
+    "id": "robofac60_123ln",
```
Comment (John-Candlebury) [inline data/json/items/gun/robofac_gun.json:429]: You need to add a migration for this one to the generic guns mod. Should be good for merging after that

**Comment by Ilysen** [comment]
generic guns migration put in. I'm not in a place where I can test right now (only have my laptop, which needs a recompile; my usual machine is occupied) so you can try it yourself or I can give it a go in a few hours

**Review by MNG-cataclysm — APPROVED** [review]

**Review by bombasticSlacks — APPROVED** [review]
Looks good to me but I haven't tested in game

**Pull request merged**